submodules/0001-VST3-base_Add-MinSizeRel-config.patch

```diff
@@ -0,0 +1,35 @@
+From 8404e348ffdca8f60b1be577d2d1d7ecfb31b73e Mon Sep 17 00:00:00 2001
+From: Matt Firth <matt.firth@bbc.co.uk>
+Date: Thu, 10 Feb 2022 16:19:57 +0000
+Subject: [PATCH] Add MinSizeRel config
+
+---
+ CMakeLists.txt | 6 ++----
+ 1 file changed, 2 insertions(+), 4 deletions(-)
```

Comment:
Do we need to patch CMAKE_CONFIGURATION_TYPES in SMTG_ConfigureCmakeGenerator.cmake as well?